src/components/video-controls/index.js

```diff
@@ -4,6 +4,7 @@ import NextChapterButton from '../next-chapter-button';
 import ProgressTimer from '../progress-timer';
 import PlaybackRates from '../playback-rates';
 import CaptionsButton from '../captions-button';
+import DownloadLinks from '../download-links';
```

Comment:
The `VideoControls` component didn't include `DownloadLinks` before (was in `index.js`)!